src/pyeval.jl

```diff
@@ -31,16 +31,13 @@ pynamespace(m::Module) =
 # and a current "file name" to use for stack traces
 function pyeval_(s::AbstractString, globals=pynamespace(Main), locals=pynamespace(Main),
                  input_type=Py_eval_input, fname="PyCall")
-    sigatomic_begin()
-    try
+    disable_sigint() do
```

Comment:
Shouldn't this be just before the `PyEval_EvalCode` line?

Reply:
I pulled out `Py_CompileString` from `disable_sigint` in d5ee7551778d3256b9dfc20a9a4b8c2494882551.  See the commit message.